### PR TITLE
Gif image support for QuickShow and File Manager

### PR DESCRIPTION
**Pull request by asliturk**

### CODE REVIEW

File: Applications/QuickShow/QSWidget.cpp [hunk 1]
```diff
@@ -84,7 +84,7 @@ void QSWidget::navigate(Directions direction)
         Core::DirIterator iterator(current_dir, Core::DirIterator::Flags::SkipDots);
         while (iterator.has_next()) {
             String file = iterator.next_full_path();
-            if (!file.ends_with(".png")) // TODO: Find a batter way to filter supported images.
+            if (!file.ends_with(".png") && !file.ends_with(".gif")) // TODO: Find a batter way to filter supported images.
```
Comment (awesomekling) [inline Applications/QuickShow/QSWidget.cpp:87]: This change doesn't seem to match the commit message?

Comment (asliturk) [reply Applications/QuickShow/QSWidget.cpp:87]: This is about navigate `gif` and `png` files in same directory.

Comment (awesomekling) [reply Applications/QuickShow/QSWidget.cpp:87]: Yes, but the commit message says "QuickShow: Use Icon class instead of Bitmap"

Comment (asliturk) [reply Applications/QuickShow/QSWidget.cpp:87]: uhhh :(

**Pull request merged**